CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+# 9.0.0-rc.3 "chocolate-chipmonk" (2019-11-20)
+
+### cdk
+
+|            |                       |
+| ---------- | --------------------- |
+| feature |  **testing:** support querying for multiple TestHarness /… ([#17658](https://github.com/angular/components/issues/17658)) ([15a5171](https://github.com/angular/components/commit/15a5171)) |
```

Comment:
"support querying for multiple TestHarness / ComponentHarness at once in locatorFor"